pkg/cmd/root.go

```diff
@@ -191,7 +191,7 @@ func (c *agentRootCommand) initialize(cmd *cobra.Command, args []string) error {
 
 	viper.WatchConfig()
 	viper.OnConfigChange(func(e fsnotify.Event) {
-		log.Debugf("Config file changed : %s", e.Name)
+		log.Tracef("Config file changed : %s", e.Name)
```

Comment:
I think this needs to be info, because its config change

Reply:
Cyrille, in another ticket, had a specific list of what should be info. This wasn't one. But maybe it should stay debug? I don't know.

Reply:
I think Info or debug would work... since @sagarkal  is right, I think stuff like config changed should be seen by customer

Reply:
put back to debug